src/quo2/components/channel_avatar.cljs

```diff
@@ -0,0 +1,50 @@
+(ns quo2.components.channel-avatar
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react-native :as rn]
+            [clojure.string :as clojure-string]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(defn channel-avatar [{:keys [big? dark? lock-status icon-color icon]}]
+  (let [locked? (= :locked lock-status)
+        lock-exists? (not= :none lock-status)]
+    [rn/view {:style {:width (if big? 32 24)
+                      :height (if big? 32 24)
+                      :border-radius (if big? 32 24)
+                      :justify-content :center
+                      :align-items :center
+                      :background-color (if dark?
+                                          colors/neutral-70
+                                          colors/neutral-30)}}
+     [rn/view {:style {:width (if big? 20 12)
+                       :height (if big? 20 12)
+                       :display :flex
+                       :justify-content :center
+                       :align-items :center}}
+      [icons/icon
```

Comment:
shouldn't the `icon/icon` component from `quo2` be used here?

Reply:
Will do now

Reply:
Done